doc/source/ray-air/examples/convert_existing_pytorch_code_to_ray_air.ipynb

```diff
@@ -1116,7 +1116,7 @@
     }
    ],
    "source": [
-    "results = batch_predictor.predict(ds, min_scoring_workers=2)"
+    "results = batch_predictor.predict(ds, batch_size=32, min_scoring_workers=2)"
```

Comment:
Manual `batch_size` required since the default `4096` is:
(1) too large of a batch size for this imagery dataset, resulting in poor parallelism and possible in OOMs with this PR,
(2) not being respected before this PR anyway, since it wouldn't batch across blocks.

Reply:
same here